src/phoenix/session/client.py

```diff
@@ -183,13 +183,13 @@ def log_evaluations(self, *evals: Evaluations, project_name: Optional[str] = Non
             table = evaluation.to_pyarrow_table()
             sink = pa.BufferOutputStream()
             headers = {"content-type": "application/x-pandas-arrow"}
-            if project_name:
-                headers["project-name"] = project_name
+            params = {"project-name": project_name}
```

Comment:
Did you test this? because I think it's _ in other places.

Reply:
yep, replacing with `project-name` everywhere